pkg/kubelet/dockershim/network/hostport/hostport_manager_test.go

```diff
@@ -372,6 +510,8 @@ func TestHostportManager(t *testing.T) {
 	for _, port := range portOpener.mem {
 		assert.EqualValues(t, true, port.closed)
 	}
+	// Clear all elements in hostPortMap
+	assert.Zero(t, len(manager.hostPortMap))
```

Comment:
that's not clearing the elements, it's asserting that someone else already did

Reply:
yeah, I wonder what I was thinking :)